src/components/multiple-post-select.js

```diff
@@ -17,6 +18,19 @@ export const MultiplePostSelect = ( { attributes, setAttributes } ) => {
 			?.map( ( { slug } ) => slug )
 	);
 
+	const [ multiplePostsState ] = useState( multiplePosts );
```

Comment:
Does this need to happen in this component? The `multiplePosts` property should available in the new component as well. To be clear, I am not saying this is wrong - just trying to grok it :)